scripts/scrape_scheduler.js

```diff
@@ -0,0 +1,18 @@
+var i = 0;
```

Comment:
Wrap this in a async/await function that returns a promise and call it from the dashboard, and replace all instances of `var` with `let` or `const`.